### PR TITLE
Pass the server type to the ol.source.ImageWMS constructor

### PR DESCRIPTION
**Pull request by fredj**
Request high DPI image from the WMS server

### CODE REVIEW

File: src/services/layerHelper.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ ngeo.LayerHelper.GROUP_KEY = 'groupName';
  * @param {string} sourceURL The source URL.
  * @param {string} sourceLayersName A dot separated names string.
  * @param {string=} opt_serverType Type of the server ("mapserver",
- *     "geoserver", qgisserver, …).
+ *     "geoserver", "qgis", …).
```
Comment (sbrunner) [inline src/services/layerHelper.js]: It's qgisserver not qgis ..., see: http://mapfish.github.io/mapfish-print-doc/layers.html#Wms%20Layer
If somewere it's not like that it's an issue ...

Comment (fredj) [inline src/services/layerHelper.js]: it's qgis not qgisserver in ol3 :-)
https://github.com/openlayers/ol3/blob/master/src/ol/source/wmssource.js

Comment (sbrunner) [inline src/services/layerHelper.js]: Then we have an issue ...

Comment (sbrunner) [inline src/services/layerHelper.js]: For me ol3 is wrong :-( 
see: http://docs.qgis.org/2.8/fr/docs/user_manual/preamble/whats_new.html#qgis-server

**Comment by ger-benjamin** [comment]
Looks good

**Comment by fredj** [comment]
Thanks for the review

**Pull request merged**